hooks/post-checkout

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# get all the updated .gitattributes files on this branch
+get_changed_attr_file() {
+    git diff --name-only "$( git merge-base master HEAD )" | grep .gitattributes
+}
+
+# scan all files in the given path and check whether the lfs file uses pointer
+check_lfs_integrity_at() {
+    files="$( git ls-files $1 )"
+
+    while read -r f; do
+        local err
+        err=false
+
+        if [[ -n "$( git check-attr filter $f | grep "filter: lfs" )" ]]; then
```

Comment:
extract this into a variable before the conditional, see the style guide: https://docs.canva.tech/style-guides/bash#use-only-simple-commands-in-conditionals